tests/testthat/test-workflow_set.R

```diff
@@ -129,6 +129,80 @@ test_that("creating workflow sets", {
   expect_true(is.null(car_set_4$result[[2]]))
 })
 
+test_that("workflow_set can handle case weights", {
+   lr_spec <- linear_reg() %>% set_engine("lm")
+   knn_spec <- nearest_neighbor() %>%
+      set_engine("kknn") %>%
+      set_mode("regression")
+
+   cars <-
+      mtcars %>%
+      dplyr::mutate(
+         wts = hardhat::importance_weights(runif(nrow(.))),
+         non_wts = runif(nrow(.))
```

Comment:
Maybe think about getting rid of randomness in the tests by using a sequence for the weights